encoding/thrift/inbound.go

```diff
@@ -209,3 +202,28 @@ func closeReader(r io.Reader) error {
 
 	return closer.Close()
 }
+
+// getBody avoids redundant copy if the transport request is already
+// io.ReaderAt compatible. If not, it creates buffer using bufferpool and returns
+// deferFn to delay bufferpool release
```

Comment:
Enhance the comment that the developer is responsible of releasing the bufferpool